src/Core/Prefab/GlobeView.js

```diff
@@ -107,7 +107,7 @@ export function createGlobeLayer(id, options) {
         }
     }
 
-    const wgs84TileLayer = new GeometryLayer(id, options.object3d || new THREE.Group());
+    const wgs84TileLayer = new GeometryLayer(id, options.object3d);
```

Comment:
Why did you changed this?

Reply:
it's changed automatically when i did rebase, i will change it back!